examples/draft-0-10-0/playground/src/index.js

```diff
@@ -1,8 +1,11 @@
+import GkManager from './GkManager';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
+console.log("Applying feature flag overwrites: ", GkManager);
```

Comment:
?

Reply:
Was thinking about logging the GKs that are applied during the playground run for debugging purposes, happy to remove this. What are your thoughts around it ?

Reply:
The value for GkManager is "window.__DRAFT_GKX", so I guess we could remove this line and relly on users checking that variable instead. What do you think ?